### ISSUE TITLE
Advance math study guide

### ISSUE THREAD

**Issue by HackyRoot** [HackyRoot/MLCC_Starter_Guide#3]
Need to add more concept with more visualization
For more details: [Prerequisites of MLCC[(https://developers.google.com/machine-learning/crash-course/prereqs-and-prework#prerequisites)

List of topics (but not limited to):

1. Algebra
- Variables, coefficients, and functions
- Linear equations
- Logarithms, and logarithmic equations
- Sigmoid function

2. Linear Algebra
- Tensor and tensor rank
- Matrix multiplication

3. Trigonometry
- Tanh (used as an activation function)

4. Statistics
- Mean, median, outliers, and standard deviation
- Histogram

5. Calculus
- The concept of a derivative
- Gradient or slope
- Partial derivatives
- Chain rule (for a full understanding of the backpropagation algorithm for training neural networks)
